string id
int32 timestamp
string meta_data

sensor_msgs/PointCloud2[] segment_clouds_map
sensor_msgs/Image[] segment_rgb_cropped_imgs
sensor_msgs/Image[] segment_depth_cropped_imgs
sensor_msgs/Image[] segment_masks
geometry_msgs/Point[] segment_centroid_poses_map
